examples/preprocessing/plot_all_scaling.py

```diff
@@ -29,8 +29,9 @@
 other in the way to estimate the parameters used to shift and scale each
 feature.
 
-``QuantileTransformer`` provides a non-linear transformation in which distances
-between marginal outliers and inliers are shrunk.
+``QuantileTransformer`` and ``PowerTransformer`` provide non-linear
+transformations in which distances between marginal outliers and inliers are
```

Comment:
While this may be true of PowerTransformer, I'm not convinced that its point is to make outliers closer, or to deal with multimodality, but to deal with skew and to otherwise make the data as gaussian as possible.

Reply:
I agree with this. Not addressed, right? But then I'll stop my nitpicking so that we can merge ;)

Reply:
Sounds good, and fixed.